src/xss-filters.js

```diff
@@ -25,14 +25,13 @@ exports._getPrivFilters = function () {
     var SENSITIVE_HTML_ENTITIES = /&(?:#([xX][0-9A-Fa-f]+|\d+);?|(Tab|NewLine|colon|semi|lpar|rpar|apos|sol|comma|excl|ast|midast);|(quot|QUOT);?)/g,
         SENSITIVE_NAMED_REF_MAP = {Tab: '\t', NewLine: '\n', colon: ':', semi: ';', lpar: '(', rpar: ')', apos: '\'', sol: '/', comma: ',', excl: '!', ast: '*', midast: '*', quot: '"', QUOT: '"'};
 
+    // TODO: CSS_DANGEROUS_FUNCTION_NAME = /(url\(|expression\()/ig;
     var CSS_UNQUOTED_CHARS = /[^%#+\-\w\.]/g,
```

Comment:
it is noted that the unquoted css filter is the whitelist filters that only the defined char will not be escaped. it is much safer from security point of view. the defined chars right now is mainly focusing on the term based on css grammar. 
```
expr
  : term [ operator? term ]*
  ;
term
  : unary_operator?
    [ NUMBER S* | PERCENTAGE S* | LENGTH S* | EMS S* | EXS S* | ANGLE S* |
      TIME S* | FREQ S* ]
  | STRING S* | IDENT S* | URI S* | hexcolor | function
  ;
```